docs/modules/3d-tiles/api-reference/tiles-3d-loader.md

```diff
@@ -59,13 +59,24 @@ const tileset3d = new Tileset3D(tilesetJson, {
   onTileLoad: tile => console.log(tile)
 });
 
-const viewport = new WebMercatorViewport({latitude, longitude, zoom, ...});
-tileset3d.update(viewport);
+const viewport = new WebMercatorViewport({
+  width: 600,
+  height: 400,
+  latitude: 40.7067584, 
+  longitude: -74.0115413, 
+  zoom: 17
+});
+tileset3d.selectTiles(viewport);
```

Comment:
Just some small tweaks here:

* Removing the `...` so there's no syntax error for a dev copy/pasting code from the docs
* Adding some sample parameters to make it easier to quick start when you're using this example